### ISSUE TITLE
Accessibility Testing for [Sitewide Homepage, Modal for redesigned homepage experience]

### ISSUE THREAD

**Issue by mmiddaugh** [department-of-veterans-affairs/va.gov-team#50684]
### Product information

- [X] Team name, product name, and feature name have been added to the title of this issue.
- [X] Team label, product label and feature label (if applicable) have been added to this issue.

### Who completed the use of color and color contrast test?

@laflannery 

### Use of color and color contrast checks

- [x] All text of 20px or smaller has a 4.5:1 contrast ratio to its background (or better)
- [x] All text of 20px or larger has a 3:1 contrast ratio to its background (or better)
- [x] Non-text elements have a 3:1  contrast ratio to their background and to neighboring elements (or better)
- [x] Color is not the only way to distinguish links from other text (eg. links are underlined)
- [x] Any charts, maps, infographics, and tables convey all information without only relying on color
- [x] Content does not refer to color, especially when providing user instructions (eg. "Click the blue button")

### Who completed the axe scans?

@laflannery 

### axe checks

- [x] Each page has been scanned using axe (results shared below)
- [ ] axe is integrated into your end-to-end testing
I do not believe these tests exist for the modal yet

### axe DevTools scan results
![image](https://user-images.githubusercontent.com/106678594/208437328-ce378b7b-39aa-469c-ba1e-5ebdd994d613.png)

### Who completed the content zoom and reflow test?

@laflannery 

### Content zoom and reflow checks

- [x] All page elements are readable and usable at 200% zoom
- [x] All page elements are readable and usable at 300% zoom
- [x] All page elements are readable and usable at 400% zoom

### How did content zoom and reflow testing go?
Update 12/21: This is now resolved on staging and I was able to proceed with testing

~~This is very difficult to do at the moment because of an existing DS bug with z-index on the modal that is resolved but not yet pushed up so I can't test yet
https://dsva.slack.com/archives/C01DBGX4P45/p1671036164833059~~

### Who completed the keyboard navigation test?

@laflannery 

### Keyboard navigation checks

- [x] Each link, button, form input, checkbox, radio button, select menu, and custom element can receive keyboard focus
- [x] Each link, button, form input, checkbox, radio button, select menu, and custom element responds to expected keys
- [x] All elements under focus have a visible focus indicator
- [x] The order of [Tab] stops made sense and was appropriate for completing tasks
